uxarray/grid.py

```diff
@@ -420,6 +420,8 @@ def build_face_edges_connectivity(self):
         mesh2_edge_node_copy, inverse_indices = np.unique(ar=mesh2_edge_nodes,
                                                           return_inverse=True,
                                                           axis=0)
+        # TODO: Make the dummy edge index in the mesh2_face_edges as INT_FILL_VALUE
```

Comment:
@philipc2 Can you help me implement this part, so that the dummy edge index in the `mesh2_face_edges` is `INT_FILL_VALUE`? Thanks